.rr.yaml

```diff
@@ -981,25 +895,25 @@ http:
     # Default: false
     redirect: true
 
-    # Path to the cert file. This option is required for SSL working.
+    # Path to the cert file. This option is required for SSL. Must not be specified if ACME is used.
```

Comment:
Example commented out, as acme is defined above and cannot be defined with key + cert.